cmd/GoBCDiceAPI/controllers/v1/version.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	"github.com/labstack/echo"
+	"github.com/raa0121/GoBCDice/cmd/GoBCDiceAPI/controllers/base"
+	"github.com/raa0121/GoBCDice/cmd/GoBCDiceAPI/helpers"
+	"github.com/raa0121/GoBCDice/cmd/GoBCDiceAPI/models"
+)
+
+type VersionController struct {
+	Router *base.BaseController
+}
+
+func (controller *VersionController) getVersion(c echo.Context) error {
+	c.Request().ParseForm()
+
+	version := models.NewVersion()
+	versionResponse := version.ToResponseMap()
+
+	return helpers.JSONResponse(c, 200, versionResponse)
```

Comment:
`version` が `Responsible` インターフェースを持っているので、`JSONResponseObject` を使うとより簡潔に書けます。

```suggestion
	return helpers.JSONResponseObject(c, 200, version)
```

Reply:
275c6c9 で直しました